.env

```diff
@@ -0,0 +1 @@
+PORT=8080
```

Comment:
Even though I'd understand this, .env is not needed in a project. You should put it in a .gitignore and also make sure that you add a .env.example

Reply:
Can you make a .env.example file that will hold this code? Your default port should already fix itself when you do not add a .env file.